packs/hpe-icsp/actions/icsp_buildplan_apply.py

```diff
@@ -0,0 +1,72 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from lib.icsp import ICSPBaseActions
+import json
+
+
+class Apply(ICSPBaseActions):
+    def run(self, buildplan_uris, server_data, connection_details):
+        if connection_details:
+            self.setConnection(connection_details)
+        self.getSessionID()
+
+        # Prepare Endpoint within ICSP API
+        endpoint = "/rest/os-deployment-jobs"
+
+        # Prepare Pesonality Data data collection
+        pload = {}
+        pload['osbpUris'] = []
+        pload['failMode'] = None
+        pload['serverData'] = []
+
+        for plan in buildplan_uris:
+            # Confirm input are integers
+            try:
+                isinstance(plan, int)
+            except:
+                raise ValueError("Build plans must be \
+                                 Integers (comma seperated)")
+
+            pload["osbpUris"].append("/rest/os-deployment-build-plans/%s"
+                                     % plan)
+
+        for server in server_data:
+            data = {}
+            pdata = {}
+            data['serverUri'] = "/rest/os-deployment-servers/%s" % server
```

Comment:
Minor style thing - usually it's a good practice to wrap format string arguments in parenthesis (e.g. `% (servers)`), this way things won't break if you add another format string parameter, but forgot to add parenthesis.